cx_plugins/file_load_plugin/src/file_load_plugin.cpp

```diff
@@ -70,6 +70,8 @@ bool FileLoadPlugin::clips_env_init(LockSharedPtr<clips::Environment> &env) {
               context->env_name_.c_str());
   for (const auto &f : init_files_) {
 
+    clips::Defmodule *main_module = clips::FindDefmodule(env.get_obj().get(), "MAIN");
```

Comment:
Can you also document this behavior in the clips-executive meta-package: https://github.com/fawkesrobotics/ros2-clips-executive/blob/master/clips_executive/doc/plugins/file_load_plugin.rst